pkg/config/config.go

```diff
@@ -18,6 +19,7 @@ var (
 	}
 
 	GlobalSettings Settings
+	FtpSettings    Settings
```

Comment:
Since ftp runs as a separate process, consider removing global variable for init.